app/serializers/project_status_serializer.rb

```diff
@@ -17,4 +24,8 @@ def show_stats?
   def name
     instance_options[:project_names][object.name]
   end
+
+  def normalized_name
+    object.name
```

Comment:
Should this be reversed with `name` and maybe send back `projecrt_names`. The word normalized doesn't make sense to me here.

Reply:
Yeah, I didn't want to disturb the existing behavior, and used "normalized" because of `normalized_licenses` - I'll change wording